Java/core/src/main/java/com/amazon/randomcutforest/sampler/CompactSampler.java

```diff
@@ -163,58 +197,75 @@ public static CompactSampler uniformSampler(int sampleSize, long seed, boolean s
     public Optional<Float> acceptSample(long sequenceIndex) {
         evictedPoint = null;
         float weight = computeWeight(sequenceIndex);
-        entriesSeen++;
         if (size < capacity) {
```

Comment:
Do we want to store the information that acceptSample was called and some weight generated, even if nothing was evicted?